src/window.rs

```diff
@@ -96,6 +96,15 @@ impl WindowBuilder {
     #[inline]
     pub fn with_transparency(mut self, transparent: bool) -> WindowBuilder {
         self.window.transparent = transparent;
+        self.window.blur = !transparent;
```

Comment:
What's the intention with the negation here?

Reply:
Transparency and blur are mutually exclusive. The negation ensures that transprency and blur are always set to the opposite of each other, so that both can never be `true` at the same time. This avoids running code paths for transparency in backends when blur is enabled (and vice versa), which in turn avoids potential bugs and improves performance.

Writing this comment, I noticed that this obviously creates a bug. When for example `with_blur(false)` is called, transparency will be enabled, which is wrong. I'm going to fix this.